[Estuary] use new version infolabels in sysinfo

ronie opened the pull request:
## Description
Update of the SysInfo window in Estuary to make use of the new System.BuildVersionCode / System.BuildVersionGit infolabels that were introduced in https://github.com/xbmc/xbmc/pull/17994.

This can go in once https://github.com/xbmc/xbmc/pull/17994 is merged.

## Screenshots (if appropriate):

before:
![sysinfo1](https://user-images.githubusercontent.com/687265/83747781-70a32500-a661-11ea-8d13-a2618bf6050c.jpg)

after:
![sysinfo2](https://user-images.githubusercontent.com/687265/83747804-7862c980-a661-11ea-81d7-7ea64ff33d3b.jpg)


## Types of change
<!--- What type of change does your code introduce? Put an `x` in all the boxes that apply like this: [X] -->
- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [X] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:
<!--- Go over all the following points, and put an `X` in all the boxes that apply like this: [X] -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [X] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

Commit:
[Estuary] use new version infolabels in sysinfo

howie-f commented:
absolutely minor: is the compiled-date behind the bottom shadow?

ksooo commented:
How does it look on systems not providing CPU usage information, like recent Android boxes?

ronie commented:
> absolutely minor: is the compiled-date behind the bottom shadow?

thx, fixed!



> How does it look on systems not providing CPU usage information, like recent Android boxes?

i've added some code to make the version section scroll up if there's no cpu info

![sysinfo3](https://user-images.githubusercontent.com/687265/83785852-330ac000-a692-11ea-86d6-200140e2207b.jpg)

howie-f commented on addons/skin.estuary/xml/SettingsSystemInfo.xml:
```diff
@@ -84,15 +84,15 @@
 				<right>50</right>
 				<top>100</top>
-				<bottom>315</bottom>
+				<bottom>340</bottom>
```
```suggestion
				<bottom>350</bottom>
```

@ronie sorry for bothering you again ;)
this change would prevent the privacy policy textbox from being glued to the line underneath:

![glued](https://user-images.githubusercontent.com/58829855/83797749-61dd6200-a6a3-11ea-9390-28cd285d18f2.png)

on top there's a gap:

![unglued](https://user-images.githubusercontent.com/58829855/83798024-dd3f1380-a6a3-11ea-89a1-c1cb0d387a4a.png)

...apart from that thumbs up

ronie replied:
> @ronie sorry for bothering you again ;)
> this change would prevent the privacy policy textbox from being glued to the line underneath:
> 
> ![glued](https://user-images.githubusercontent.com/58829855/83797749-61dd6200-a6a3-11ea-9390-28cd285d18f2.png)

people who take the time and effort to test my pr's can bother me as much as they want ;-)

though i'm unsure what problem you're referring to here...
the screenshot looks ok to me?

> on top there's a gap:
> 
> ![unglued](https://user-images.githubusercontent.com/58829855/83798024-dd3f1380-a6a3-11ea-89a1-c1cb0d387a4a.png)

true that. but it's also like that in the current version of estuary.
will fix that as well.

howie-f replied:
yeah, the gap at top of the textbox is correct. if the text starts scrolling you can see that the gap at the _bottom_ is missing

ronie replied:
ok, i've re-added the gaps. both at the top & bottom.

DaVukovic commented:
Would it be possible to make that section as long as the text above?

![image](https://user-images.githubusercontent.com/7235787/83936537-dc51d300-a7c4-11ea-9eaa-80f397636d72.png)

howie-f commented:
> Would it be possible to make that section as long as the text above?

ideally the length of the progress-bar would always match the length of the label... don't know if that is achievable

DaVukovic commented:
>ideally the length of the progress-bar would always match the length of the label... don't know if that is achievable

That would be cool, but I guess at some point we might have cut the line (at least for the text). The current cpu-info text layout would probably allow to show the text for 10 cores. Then the line would exceed the width of the screen. If you then think of 12 or 16 core-CPUs, then we have do it somehow different. Not sure what the best approach would be then, as we had that discussion earlier ;).
Just thought that it would be cool if the bar for the CPU-usage would match the length of the text

howie-f commented:
> as we had that discussion earlier ;)

yo, followed that discussion.. don't worry, these are only suggestions - no bikeshedding. as we all know: ask 10 people, get 100 opinions - like always in life

jjd-uk commented:
> > Would it be possible to make that section as long as the text above?
> 
> ideally the length of the progress-bar would always match the length of the label... don't know if that is achievable

I don't believe that's possible to autosize those bars. Assuming it's not, then as we've now nothing to the right you could just make those bars full width, apart from the margins either side. Like this

![image](https://user-images.githubusercontent.com/5781142/83945581-99fbb680-a803-11ea-9f59-82b4036a95f9.png)

Just needs `<width>730</width>` changing to `<right>30</right>` giving whatever value margin you want from the right.

howie-f commented:
@jjd-uk maybe the only possible option, two bars with different width might be even worse. imho

ronie commented:
> ![image](https://user-images.githubusercontent.com/5781142/83945581-99fbb680-a803-11ea-9f59-82b4036a95f9.png)

to me it doesn't look that visually pleasing to have such a wide bar.
i'll stick with what we have now.
